ch2driver/pytpcc/runtime/loader.py

```diff
@@ -184,8 +196,13 @@ def loadWarehouse(self, w_id):
                 o_tuples.append(o_tuple)
                 h_amount = h_tuples[cIdPermutation[o_id-1]-1][6]
                 if not newOrder:
-                    c_tuples[cIdPermutation[o_id-1]-1][16] = total_ol_amount - h_amount
-                    c_tuples[cIdPermutation[o_id-1]-1][17] = h_amount
+                    if self.schema == constants.CH2_DRIVER_SCHEMA["CH2"]:
+                        c_tuples[cIdPermutation[o_id-1]-1][16] = total_ol_amount - h_amount
+                        c_tuples[cIdPermutation[o_id-1]-1][17] = h_amount
+                    else:
+                        c_tuples[cIdPermutation[o_id-1]-1][7] = total_ol_amount - h_amount
+                        c_tuples[cIdPermutation[o_id-1]-1][8] = h_amount
```

Comment:
A suggestion to make this kind of thing maybe clearer would be to write it something like this:
```
if self.schema == constants.CH2_DRIVER_SCHEMA["CH2"]:
    c_balance_idx, c_ytd_payment_idx = 16, 17
else:
    c_balance_idx, c_ytd_payment_idx = 7, 8

c_tuples[cIdPermutation[o_id-1]-1][c_balance_idx] = total_ol_amount - h_amount
c_tuples[cIdPermutation[o_id-1]-1][c_ytd_payment_idx] = h_amount
```
Bit less repetitive, a bit clearer what fields are being updated

Reply:
Done, for this and other similar instances.